assets/src/edit-story/elements/media/scalePanel.js

```diff
@@ -71,14 +72,26 @@ const ResetButton = styled.button`
   padding: 1px 8px 0 8px;
 `;
 
+const ACCEPTED_KEYS = [
+  'ArrowUp',
+  'ArrowDown',
+  'ArrowLeft',
+  'ArrowRight',
+  'PageUp',
+  'PageDown',
+];
+
 function ScalePanel({ setProperties, width, height, x, y, scale }) {
+  // Handle different step size for mouse and keyboard users
+  const usingKeyboard = useIsUsingKeyboard(null, ACCEPTED_KEYS);
```

Comment:
Not sure how soon this will start causing issues. It's likely that we will be adding all kinds of shortcuts, such as: do `10` by default and `1` when <kbd>shift</kdb> is pressed - that's consistent with the how we move elements on the canvas. Maybe it'd be better to use plain key handler (or `useKeyDown`)? We should be able to cancel the default behavior and implement our own scrubbing.

Reply:
It looks like using `shift` was dismissed as per the discussions on the original issue, but let me know if that's still needed